test/xds/xds_server_integration_test.go

```diff
@@ -111,13 +91,31 @@ func setupGRPCServer(t *testing.T, bootstrapContents []byte) (net.Listener, func
 	if err != nil {
 		t.Fatal(err)
 	}
-
-	// Initialize an xDS-enabled gRPC server and register the stubServer on it.
+	stub := &stubserver.StubServer{
+		EmptyCallF: func(ctx context.Context, in *testpb.Empty) (*testpb.Empty, error) {
+			return &testpb.Empty{}, nil
+		},
+		UnaryCallF: func(ctx context.Context, in *testpb.SimpleRequest) (*testpb.SimpleResponse, error) {
+			return &testpb.SimpleResponse{}, nil
+		},
+		FullDuplexCallF: func(stream testgrpc.TestService_FullDuplexCallServer) error {
+			for {
+				_, err := stream.Recv() // hangs here forever if stream doesn't shut down...doesn't receive EOF without any errors
+				if err == io.EOF {
+					return nil
+				}
+			}
+		},
+	}
+	// Initialize an xDS-enabled gRPC server and use the helper to start the test service.
```

Comment:
same comment here. Just modify the existing statement instead of writing a new line